### ISSUE TITLE
Confused about the results

### ISSUE THREAD

**Issue by fufrank5** [thunlp/OpenNRE#52]
I ran the the commands in sequence:
`python train.py --model_name MODEL_NAME`
`python test.py --model_name MODEL_NAME`
`python draw_plot.py --model_name MODEL_NAME`

I got:
cnn_max-auc=0.3929
cnn_att-auc=0.3801
pcnn_att-auc=0.4008
birnn_att-auc=0.3771

The results shows that ATT is worse than MAX, which is wired.
BIRNN is worse than PCNN and about the same as CNN, which contradicts previous claim and graph in the old version of this repository.

The results of PCNN seems to be significantly better than (Zeng et al.,2015) reported in the paper.
Is this because this program is directly picking the best epoch on the test set?
The paper does not specify how they select the final model for prediction. (Lin et al. 2016) used epoch-25.

I'm not familiar with the dataset. I hope someone with knowledge can explain this.
Thanks.

**Comment by gaotianyu1350** [comment]
Thanks for your feedback! There was a bug in the attention module which led to the abnormal results. Now the bug has been fixed and you could check our latest update.

**Comment by fufrank5** [comment]
I think there there might be a wrong version of training data posted. There are 570088 sentences in train.txt  instead of 522611. There's also an overlap between train.txt and test.txt.
The data from https://github.com/thunlp/NRE seems to be consistent with the papers. Is this the correct version? Thanks.
